### DISCUSSION TITLE
Occasional missing images in "build" folder when rebuilding APKs from inside another process

### DISCUSSION THREAD

**Discussion by ericcdub** [iBotPeaches/Apktool#3469] [Q&A]
I'm investigating an issue whereby some images are sometimes missing from the "build" folder created when rebuilding an APK. The set of missing images varies from one rebuild to another. I can confirm that all the images are present in the folder passed to Apktool.
I'm running Apktool.bat from within another process (our proprietary localisation software) using the Windows CreateProcess API function. It occasionally reproduces when running automated test scripts on a machine with 8GB of RAM but never when used on dev machines (all have 16GB). I didn't want to log a bug proper as I don't have enough information but was wondering if this is something you've seen before or might have an explanation for?

**Comment by iBotPeaches** [comment]
Are you using aapt1 or aapt2? It it more reproducible with one over the other?

aapt2 is a multi stage build, while aapt1 isn't which is why I ask.

**Comment by ericcdub** [comment]
Hi,

We've been using aapt1 (I only learned of the aapt2 flag when you posted! :) 
We're going to switch over to aapt2 to see if that addresses the problem. I'm not sure if it's some kind of race condition in how we call ApkTool, or if something is removing files on the host machine like antivirus, but I don't think it's that either.
